ansible/roles/couchdb2/tasks/main.yml

```diff
@@ -83,3 +80,26 @@
   tags:
     - cron
     - backup
+
+- name: start the couchdb2 service
+  service: name="couchdb2" state=started enabled=yes
+  tags: after-reboot
+
+- name: couchdb2 monit config
+  sudo: yes
+  template:
+    src: "monit.couchdb2.j2"
+    dest: "/etc/monit/conf.d/couchdb2"
+    group: root
+    owner: root
+    mode: 0640
+  tags: monit
+
+- name: restart monit
+  service: name="monit" state=reloaded
+  tags: monit
```

Comment:
I think this should be a handler and not a task

Reply:
using handlers now...